### PR TITLE
Spinnable Wheel

### PR DESCRIPTION
**Pull request by zfox23**
It's a wheel that spins. Nick wants it. Now everyone can have it. Woohoo!

### CODE REVIEW

File: usefulUtilities/spinnableWheel/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Spinnable Wheel
+When you rez `spinnableWheel.json` in world, you'll see a...spinnable wheel. Click or user your hand controller laser on the wheel to spin it. Modify the User Data of the wheel and reload the Wheel's server script to update the options that appear on the wheel.
```
Comment (MiladNazeri) [inline usefulUtilities/spinnableWheel/README.md]: lol I always love things like that

Comment (MiladNazeri) [reply usefulUtilities/spinnableWheel/README.md]: Click or user your hand controller laser on the wheel to spin it. <-- typo?

File: usefulUtilities/spinnableWheel/spinnableWheelServer.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+/* global console */
+
+(function() {
+    var that;
+    var CHECK_SPINNING_TIMEOUT_MS = 100;
+    var RAD_PER_DEG = Math.PI / 180;
+    var SMALL_Z_ANGULAR_VELOCITY_RAD_PER_SEC = 15 * RAD_PER_DEG;
+    var MIN_ANGULAR_VELOCITY_RAD_PER_SEC = 40;
+    var MAX_ANGULAR_VELOCITY_RAD_PER_SEC = 100;
+
```
Comment (MiladNazeri) [inline usefulUtilities/spinnableWheel/spinnableWheelServer.js]: Can you add some function comments?

**Comment by salinacsoto** [comment]
bitmoji

File: usefulUtilities/spinnableWheel/README.md (hunk 1 above)
Comment (zfox23) [reply usefulUtilities/spinnableWheel/README.md]: yes indeed, will fix :)

File: usefulUtilities/spinnableWheel/spinnableWheelServer.js (hunk 2 above)
Comment (zfox23) [reply usefulUtilities/spinnableWheel/spinnableWheelServer.js]: yeahhhhh i can, thank you for calling it out :)

**Pull request merged**